index.html

```diff
@@ -971,6 +999,91 @@ <h2>The <code>application/credential-claims-set-1.1+json</code> Media Type</h2>
   <div class="issue" data-number="42"></div>
 </section>
 
+<section>
+  <h2>Appendix</h2>
+  <section>
+    <h2>Example Mapping</h2>
+    <p>
+The following describes a mapping from <code>application/vc+jwt</code> to
+<code>application/vc+ld+json</code>. This is one possible unidirectional mapping
+between 2.0 VC-JWTs and the VC Data Model; other such mappings are possible.
+    </p>
```

Comment:
This section runs afoul of the resolution made at the most recent VCWG F2F Meeting, namely that "Transformation rules MUST be defined", specifically, I expect that a number of the individuals in the WG presumed that the intent here was to have normative language defined somewhere that was testable. The language in this section isn't normative and therefore won't be tested by the WG.

It is imperative that if the WG is defining transformation rules, that those rules be testable and demonstrated to provide interoperability at some layer. Not providing that assurance (demonstrable interoperability) with a specification being produce by the WG would be a failure of our remit.

https://www.w3.org/2017/vc/WG/Meetings/Minutes/2023-02-16-vcwg#resolution1

This section needs to be something that has normative language, is tested during CR, and results in measurable demonstrations of multi-vendor interoperability.